types/src/chainspec/genesis_config.rs

```diff
@@ -358,7 +370,9 @@ impl From<&Chainspec> for GenesisConfig {
             .with_round_seigniorage_rate(chainspec.core_config.round_seigniorage_rate)
             .with_unbonding_delay(chainspec.core_config.unbonding_delay)
             .with_genesis_timestamp_millis(genesis_timestamp_millis)
+            .with_gas_hold_balance_handling(gas_hold_balance_handling)
             .with_gas_hold_interval_millis(gas_hold_interval_millis)
+            .with_gas_hold_balance_handling(gas_hold_balance_handling)
```

Comment:
There should only be one of these, no?

Reply:
looks like a merge artifact btwn alex and I. its harmless, but wrong. I'll put up a commit removing it momentarily.